### ISSUE TITLE
[Feature Request]suggest to add regularization in policy network

### ISSUE THREAD

**Issue by zhaopansong** [DLR-RM/stable-baselines3#240]
**Important Note: We do not do technical support, nor consulting** and don't answer personal questions per email.
Please post your question on [reddit](https://www.reddit.com/r/reinforcementlearning/) or [stack overflow](https://stackoverflow.com/) in that case.

### 🚀 Feature

just give open api for setting tf network regularization parameters. Such as 
tf.keras.regularizers.L1(0.3)  # L1 Regularization Penalty
tf.keras.regularizers.L2(0.1)  # L2 Regularization Penalty
tf.keras.regularizers.L1L2(l1=0.01, l2=0.01)  # L1 + L2 penalties

### Motivation

Please outline the motivation for the proposal.
Is your feature request related to a problem? e.g.,"I'm always frustrated when [...]".
If this is related to another GitHub issue, please link here too.

### Alternatives

A clear and concise description of any alternative solutions or features you've considered, if any.

### Checklist

- [ ] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo (**required**)

**Comment by Miffyli** [comment]
Please fill on the template next time. Also note that this is PyTorch-based repo, not Tensorflow.

Parameter regularization is not too common of a thing outside e.g. generalization over multiple tasks (e.g. ProcGen), and as such I am not sure about adding it as a default option. You can use optimizer's `weight_decay` parameter to achieve L2 loss (e.g. `A2C(optimizer_kwargs={"weight_decay": 1e-5})`), or by defining [custom policies](https://stable-baselines3.readthedocs.io/en/master/guide/custom_policy.html). I would say better access to for these features could go in v1.1+.
